src/desktop/apps/experimental-app-shell/apps/artwork/artworkClient.tsx

```diff
@@ -11,9 +11,12 @@ export const artworkClient = () => {
   const ViewInRoomView = require("desktop/components/view_in_room/view.coffee")
   const $ = require("jquery")
   const mediator = require("desktop/lib/mediator.coffee")
+  const pageType = window.location.pathname.split("/")[1]
 
-  const artworkSlug = location.pathname.replace(/\/artwork\//, "")
-  recordArtworkView(artworkSlug, sd.CURRENT_USER)
+  if (pageType === "artwork") {
```

Comment:
Interesting, does this wind up firing as you navigate (to an artwork, when starting your session on a non-artwork page)?

Looks like the other things here (`mediator.on` hooks) just need to be called once in your session, in order to successfully be registered. Whereas this `recordArtworkView` should be called potentially on every navigation?

(sidenote- this is a mutation to record a view, firing on render. prob an oversight it's not in Reaction)

Reply:
This was how you had it before -- i had revised the inside of this in a previous PR and only mounted the client bits if it was an exactly page match on SSR but realized last night that we need this client stuff mounted immediately as we can navigate to the page later. So I just re-added this how it was.